### PR TITLE
Component/counter

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
create a counter component which can increment and decrement in custom intervals

### CODE REVIEW

**Review by lajanzen — APPROVED** [review]

File: src/app/components/Counter/Counter.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  align-self: center;
+  justify-self: center;
+  border: none;
+  width: 20px;
+  height: 10px;
+  transform: translateY(-50%);
+  cursor: pointer;
+`
+
+const ValueView = styled.p``
```
Comment (iva585) [inline src/app/components/Counter/Counter.tsx:50]: I don't understand why you created a styled component for an element without styling it? 🧐

File: src/app/Icons/IconWrapper.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default function IconWrapper({
 }
 
 const Wrapper = styled.div<Partial<WrapperProps>>`
-  width: ${({ width }) => (width ? `${width}px` : '24px')};
+  width: ${({ width }) => (width ? `${width}px` : '100%')};
```
Comment (iva585) [inline src/app/Icons/IconWrapper.tsx:17]: cool :)

File: src/app/components/Counter/Counter.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  onIncrement,
+  onDecrement,
+}: CounterProps): JSX.Element {
+  return (
+    <CounterContainer>
+      <VoteButton displayType="up" onClick={onIncrement}>
+        <ChevronUp />
+      </VoteButton>
+      <ValueView>{value}</ValueView>
+      <VoteButton displayType="down" onClick={() => value > 0 && onDecrement()}>
```
Comment (iva585) [inline src/app/components/Counter/Counter.tsx:23]: Why does the value have to be greater than zero? :D interesting, maybe we could talk about how this type of collective decision making works STS :)

**Review by iva585 — COMMENTED** [review]
LGTM, very interested in how the app works :) 
The one thing I don't understand in terms of code is why you created a styled component for the p? Are you planning on styling it later? If so, it's an unnecessary line of code rn IMO.

File: src/app/components/Counter/Counter.tsx (hunk 1 above)
Comment (F-Kirchhoff) [reply src/app/components/Counter/Counter.tsx:50]: thanks thats not necessary anymore.

File: src/app/components/Counter/Counter.tsx (hunk 3 above)
Comment (F-Kirchhoff) [reply src/app/components/Counter/Counter.tsx:23]: oh the counter should not go below zero thats all.

**Pull request merged**